### PR TITLE
Hack up survey JSON editor for debugging

### PR DESCRIPTION
**Pull request by gino-m**
Hacking up an editor to help us debug, tweak, backup, and restore our survey definitions.

image

@DaoyuT PTAL?

### CODE REVIEW

File: web/src/app/services/data-store/data-store.service.ts [hunk 1]
```diff
@@ -66,6 +66,27 @@ export class DataStoreService {
...
+      await firstValueFrom(
+        this.db.collection(SURVEYS_COLLECTION_NAME).doc(id).get()
+      )
+    ).data();
+  }
+
+  /**
+   * Updates the raw survey object in the db. Used for debbuging only.
+   */
+  async saveRawSurvey(id: string, data: any) {
```
Comment (DaoyuT) [inline web/src/app/services/data-store/data-store.service.ts:83]: nit: `data: any` <--- Could this be string?

**Review by DaoyuT — CHANGES_REQUESTED** [review]
We should have test for this new component ideally, but given this is a non-prod tool, I'll let it slide. Left one nit, LGTM other than that.

File: web/src/app/services/data-store/data-store.service.ts (hunk 1 above)
Comment (gino-m) [reply web/src/app/services/data-store/data-store.service.ts]: How about this?

**Review by DaoyuT — APPROVED** [review]
LGTM, there seem to be a lint issue which is failing the presubmit

**Review by DaoyuT — APPROVED** [review]

**Pull request merged**